src/services/blocks/BlocksService.ts

```diff
@@ -448,8 +448,18 @@ export class BlocksService extends AbstractService {
 				) {
 					// multiSig.asMulti.args.call is an OpaqueCall (Vec<u8>) that we
 					// serialize to a polkadot-js Call and parse so it is not a hex blob.
-					const call = registry.createType('Call', argument.toHex());
-					newArgs[paramName] = this.parseGenericCall(call, registry);
+					try {
+						const call = registry.createType(
+							'Call',
+							argument.toHex()
+						);
+						newArgs[paramName] = this.parseGenericCall(
+							call,
+							registry
+						);
+					} catch {
+						newArgs[paramName] = argument;
```

Comment:
Would be good to log a warning here maybe? Or is there a valid scenario where users pass in a bag of bytes that we're not supposed to understand how to parse?

Can we make `parseGenericCall` return a richer error type and use `catch(err)` to present something that makes sense to users?

Reply:
I think logging a message to stdout could be ok, but I would think many users are not checking logs unless the endpoint returns 400 + codes.

We could also put something into the response noting the bytes are not decodable. However, I am not sure where that message would go. Right now the response maps to exactly in the block, and in this case we would need to add a field somewhere. I also think adding the field right here could be strange since these are supposed to be the arguments to extrinsic. 

A good middle ground could be putting a note in the docs that if you are expecting a decoded OpaqueCall and you instead get an encoded one it is likely that it is not decodable. The user could then try decoding themselves.

Reply:
Ok added docs